docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -18,29 +18,29 @@ An image that {ProductName} builds from source code in a repository. One or more
 == Conftest 
 A utility for testing structured configuration data. Use Conftest to validate container information.
 
-== Enterprise Contract (EC) 
-A set of release policies that you apply to your managed environment. Use the EC to prevent the release of Supply chain Levels for Software Artifacts (SLSA) that are non-compliant. 
+== Enterprise contract (EC) 
+A set of release policies that you apply to your managed environment. Use the EC to prevent releases that are not compliant with Supply-chain Levels for Software Artifacts (SLSA) guidelines. 
```

Comment:
Please note that although we'd normally do "supply chain," this specific term is named "Supply-chain"